CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.5.0-beta.2
+- Added support for Inline Arrays in Struct.
```

Comment:
Inline Arrays -> inline arrays  (It's not two names, so they don't need to be capitalized).
Struct -> structs -> `Struct`s.